pkg/webhook/admission/builder/builder.go

```diff
@@ -73,10 +73,19 @@ func (b *WebhookBuilder) Name(name string) *WebhookBuilder {
 	return b
 }
 
-// Type sets the type of the admission webhook
-// This is required
-func (b *WebhookBuilder) Type(t types.WebhookType) *WebhookBuilder {
-	b.t = &t
+// Mutating sets the type to mutating admission webhook
+// Only one of Mutating and Validating can be invoked.
+func (b *WebhookBuilder) Mutating() *WebhookBuilder {
+	m := types.WebhookTypeMutating
+	b.t = &m
+	return b
+}
+
+// Validating sets the type to validating admission webhook
+// Only one of Mutating and Validating can be invoked.
+func (b *WebhookBuilder) Validating() *WebhookBuilder {
+	m := types.WebhookTypeValidating
```

Comment:
do we have a default?  Can we have a default?

Reply:
No, I don't think we can choose a reasonable defaulting for webhook type.